README.md

```diff
@@ -183,28 +183,31 @@ Mounting the Engine will provide the basic routes to authenticating a shop with
 |POST    |'/webhooks/:type'              |Webhook Callback              |
 
 
-The default routes of the Shopify rails engine, which is mounted to the root, can be altered to mount on a different route. The `config/routes.rb` can be modified to put these under a nested route (say `/app-name`) as:
+If required the engine can be mounted at a nested route, eg:
 
 ```ruby
 mount ShopifyApp::Engine, at: '/nested'
 ```
 
-This will create the Shopify engine routes under the specified subpath, as a result it will redirect new consumers to `/nested/login`. If you mount the engine at a subpath you'll also need to update the omniauth initializer to include a custom `callback_path` e.g:
+This will create the Shopify engine routes under the specified subpath. You'll also need to make some updates to your `shopify_app.rb` and `omniauth.rb` initializers.
+
+update the shopify_app initializer to include a custom `root_url` and `login_url` e.g:
```

Comment:
We should remove `login_url` from the documentation (here and in the lines below)